.github/workflows/maven.yml

```diff
@@ -26,6 +26,7 @@ on:
       - development
       - master
       - lineageondemand
+      - taskdg1924
```

Comment:
Rebase w beta to resolve the `maven.yml` conflict.

Reply:
done